packages/core/tsconfig.build.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.build.json",
 
   "compilerOptions": {
-    "outDir": "./build"
+    "outDir": "./build",
+    "lib": ["dom"]
```

Comment:
We have tried very hard to not introduce any environment specific stuff into core, and if possible I'd like to keep it that way.

Reply:
I see it's to find JsonWebKey. i'd prefer that you define JsonWebKey in sd jwt as a custom type and not depend on lib dom

Reply:
Understood. Actually we have an open [PR](https://github.com/openwallet-foundation-labs/sd-jwt-js/pull/130) for this issue in our repo. I'll let you know when it's resolved. :)

Reply:
@cre8 @TimoGlastra After some digging, I found out that we can get rid of error by adding this option in tsconfig.json

```json
{
  "skipLibCheck": true
}
```

Reply:
We added jsonwebkey to the repo, so the Dom reference is not required anymore

Reply:
I'll update the PR :)

Reply:
It's updated